### ISSUE TITLE
Always use Qt client instead of GTK client for upcoming 4.0 release

### ISSUE THREAD

**Issue by Pentaphon** [flathub/com.transmissionbt.Transmission#44]
According to the draft changelog, it seems 4.0 has moved significantly more forward with the Qt client than it has with the GTK client. 

https://github.com/transmission/transmission/blob/refactor/update-news/news/news-4.0.0-beta-1.md

Perhaps 4.x users may benefit from using only the Qt client on all platforms, even if they are using GTK desktops.

**Comment by Pentaphon** [comment]
> Do you have examples for this claim?

No. I was wondering since the Flatpak originally came as a Qt app for me and then switched to GTK at some point, which doesn't look as good as the Qt app.

**Comment by wjt** [comment]
This Flatpak has always been the GTK version.

**Comment by Pentaphon** [comment]
> This Flatpak has always been the GTK version.

Ignore my last post. I was talking about this in another discussion and accidentally posted it here.

**Comment by mikelpr** [comment]
> > Perhaps 4.x users may benefit from using only the Qt client on all platforms, even if they are using GTK desktops.
> 
> Do you have examples for this claim? Switching to Qt will possibly introduce regressions with folder selections in the application's settings. Just search for `QFileDialog` and `Flatpak`, and you'll find enough examples.

the Qt version of Transmission is much more feature complete and can control remote transmission instances so its pluses outweigh its cons, and QFileDialog - Flatpak interactions absolutely have to be fixed and must not be something to consider at app level when the app is moving towards Qt and dropping GTK - instead of preserving the dying GTK build just because the toolkit has better flatpak support, we need to push for Qt to better support flatpak

**Comment by o-alquimista** [comment]
> According to the draft changelog, it seems 4.0 has moved significantly more forward with the Qt client than it has with the GTK client.
> 
> https://github.com/transmission/transmission/blob/refactor/update-news/news/news-4.0.0-beta-1.md
> 
> Perhaps 4.x users may benefit from using only the Qt client on all platforms, even if they are using GTK desktops.

That URL is now 404. The final release notes for v4.0 are [here](https://github.com/transmission/transmission/releases/tag/4.0.0), and it shows that both Qt and GTK clients have received fixes and modernization.

With that said, I think it would be nice to have both Qt and GTK clients available from Flathub.

**Comment by ghost** [comment]
I have tested the QT version, and it was really good. Unfortunately, due to submission rules, I couldn't create a QT version for it.

> **Duplicate submissions**
> 
> Multiple submissions of the same application (in terms of functionality, content, visuals, or user experience) each built with different frameworks or toolkits will not be accepted. Multiple submissions of a forked app with minimal changes in the fork (in terms of functionality, content, visuals, or user experience) will not be accepted.

https://docs.flathub.org/docs/for-app-authors/requirements#duplicate-submissions

**Comment by fatmalama** [comment]
@yakushabb maybe an exception for could be asked. the GTK client is quite useless without remote management support imho

**Comment by wjt** [comment]
Feel free to take over this Flatpak and do whatever is desired, e.g. transform it into the Qt version in an update. Per #105 and the linked forum post from 2021 I do not want to maintain it any longer!

**Comment by fatmalama** [comment]
@wjt oh sorry I wasn't aware haha

**Comment by fatmalama** [comment]
@yakushabb I'm prepping a Qt manifest. I'll send you a PR as soon as I have clarity on what to do about its missing appstream https://github.com/transmission/transmission/issues/8862

**Comment by ghost** [comment]
Please don't bother with it. We need to consider users' behaviours. Not just the GUI/Toolkit, you also need to change the runtime version, which would inflict a lot of headaches for users.

**Comment by ghost** [comment]
See also: https://discourse.flathub.org/t/what-if-new-kind-of-repositories/11840

**Comment by fatmalama** [comment]
@yakushabb alright, do consider however the GTK version is not at feature parity with the Qt version in a very important way (managing remote daemons) which is a huge headache

**Comment by o-alquimista** [comment]
> > **Duplicate submissions**
> > Multiple submissions of the same application (in terms of functionality, content, visuals, or user experience) each built with different frameworks or toolkits will not be accepted. Multiple submissions of a forked app with minimal changes in the fork (in terms of functionality, content, visuals, or user experience) will not be accepted.
> 
> https://docs.flathub.org/docs/for-app-authors/requirements#duplicate-submissions

There are now two Thunderbird flatpaks in Flathub, one for [ESR](https://flathub.org/en/apps/org.mozilla.thunderbird_esr) and another for [latest release](https://flathub.org/en/apps/org.mozilla.thunderbird). Either the rules have changed and the docs are outdated, or Mozilla is violating them.

**Comment by ghost** [comment]
Since I'm not a fan or supporter of the rule, you can open an issue in the related repository: https://github.com/flathub/flathub/

I wish they would make exceptions for some mature projects.

**Comment by bztd** [comment]
In my experience with X11, the package works well except for the issue I reported to the development team regarding a menu that remains active. If the development team prioritizes Qt, the package should be moved to that environment; I don't think there would be widespread dissatisfaction. I don't believe the idea of ​​having two branches for the package will be successful.
